src/cli/databases_create.rs

```diff
@@ -105,7 +124,7 @@ fn clusters_create(
                 payload: serde_json::to_string(&json)
                     .map_err(|e| serialize_error(e.to_string(), span))?,
             },
-            Instant::now().add(control.timeout()),
+            Instant::now().add(timeout),
```

Comment:
Should this be using deadline rather than timeout?

Reply:
Yes it should, done